kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Misc.kt

```diff
@@ -0,0 +1,30 @@
+@file:JvmMultifileClass
+@file:JvmName("MiscKt")
+
+package com.kotlindiscord.kord.extensions.utils
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import mu.KLogger
+import mu.KotlinLogging
+
+/**
+ * Neutral and default logger.
+ */
+public fun classLogger(): KLogger = KotlinLogging.logger() {}
```

Comment:
I made a comment about this earlier, but I really don't see the point of doing this. It doesn't add any utility, and currently your loggers aren't being instantiated in the right scope. It's better to just do this as explained in my other comment.